experiment/contractNipopow.sol

```diff
@@ -354,13 +373,9 @@ contract Crosschain {
             "Contesting proof[1:] is not different from existing[lca+1:]"
         );
 
-        // We can ask the caller to provide the level for their proof
         require(
             bestArg(existingHeadersHashed, lca) <
-                bestArg(
-                    contestingHeadersHashed,
-                    contestingHeadersHashed.length - 1
-                ),
+                bestArgAtLevel(contestingHeadersHashed, bestLevel),
```

Comment:
This is no longer a *best* arg at level, it's just the arg.

Reply:
Done